{{ cookiecutter.repo_name }}/.gitignore

```diff
@@ -55,7 +55,7 @@ coverage.xml
 *.log
 
 # Sphinx documentation
-docs/_build/
+docsrc/_build/
```

Comment:
`docsrc`? This name is a little strange to me.

Reply:
Never mind. It makes sense after reading the README. Not sure what's standard here.